mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/TaskExecutor.java

```diff
@@ -126,9 +130,22 @@ public TaskExecutor(
         MachineDefinition machineDefinition =
             MachineDefinitionUtils.sys(workerPorts, workerConfiguration.getNetworkBandwidthInMB());
         String hostName = workerConfiguration.getExternalAddress();
+        Map<String, String> envMap = new HashMap<>(System.getenv());
+        // Adding a default id for back-compat support.
+        envMap.putIfAbsent(WorkerConstants.WORKER_CONTAINER_DEFINITION_ID, "defaultContainerId");
+
         this.taskExecutorRegistration =
-            new TaskExecutorRegistration(
-                taskExecutorID, clusterID, getAddress(), hostName, workerPorts, machineDefinition);
+            TaskExecutorRegistration.builder()
+                .machineDefinition(machineDefinition)
+                .taskExecutorID(taskExecutorID)
+                .clusterID(clusterID)
+                .hostname(hostName)
+                .taskExecutorAddress(getAddress())
+                .workerPorts(workerPorts)
+                .taskExecutorAttributes(ImmutableMap.copyOf(envMap))
```

Comment:
Hmm. I'm unsure if we need to send the entire env map to the mantis master. Instead, can we look for **a** env variable that can somehow be serialized/deserialized as a `Map<String, String>`? For example, can we put all the attributes under the env var `TASK_EXECUTOR_ATTRIBUTES`?